tasks/copy_to_glacier/test/unit_tests/test_copy_to_glacier.py

```diff
@@ -331,6 +331,85 @@ def test_task_overridden_multipart_chunksize(self):
         self.assertEqual(expected_granules, result["granules"])
         self.assertIsNone(config_check.bad_config)
 
+    @patch.dict(
```

Comment:
May want a test to validate that default is taken if override not set as well.

Reply:
I treated the other tests as base cases, and this as testing the override.